src/firefly/js/ui/UploadTableChooser.js

```diff
@@ -214,32 +213,31 @@ const LoadedTables= (props) => {
     if (data.length === 0) {
         return <NoTables/>;
     }
-    const columns = [{name: 'Table Title', width: 45}, {name: 'No. of Cols', width: 15}, {name: 'No. of Rows', width:15},
+    const columns = [{name: 'Table Title', width: 40}, {name: 'No. of Cols', width: 15}, {name: 'No. of Rows', width:15},
         {name: 'tblId', visibility: 'hidden'}];
     const highlightedRow = 0; //default selection
     const tbl_id = 'existing-table-list';
 
     const tableModel = {tbl_id, tableData: {columns, data}, highlightedRow, totalRows: data.length};
 
     return (
-        <div style={{margin: '10px', position: 'relative', width: '100%', display: 'flex', alignItems: 'stretch', flexDirection: 'column'}}>
-            <div style={{color: 'gray', fontSize: 'larger', lineHeight: '1.3em'}}>
+        <Box sx={{ width: 1, height: 1, display: 'flex', flexDirection: 'column'}}>
```

Comment:
`display: 'flex', flexDirection: 'column'` is a Stack.  Should use that instead.
```jsx
<Stack width={1} height={1}>
```